studio/tests/pages/projects/logs-query.test.js

```diff
@@ -177,23 +177,15 @@ test('Upgrade prompt displays if user on free plan and tries to view beyond 24hr
   useProjectSubscription.mockReturnValue({
     subscription: {
       tier: {
-        supabase_prod_id: 'tier_pro',
+        supabase_prod_id: 'free',
```

Comment:
also, the mock will already be returning `tier_free` (we reset the mocks at the top level setup callback)

Reply:
it'll be easier to place the mock at the top of the test, before the render. Because otherwise the cached value will be from the first call of the mock hook, before you replace the mock.